src/xenia/kernel/xam/xam_enum.cc

```diff
@@ -82,13 +82,24 @@ dword_result_t XamEnumerate(dword_t handle, dword_t flags, lpvoid_t buffer,
 }
 DECLARE_XAM_EXPORT1(XamEnumerate, kNone, kImplemented);
 
-dword_result_t XamCreateEnumeratorHandle(unknown_t unk1, unknown_t unk2,
-                                         unknown_t unk3, unknown_t unk4,
-                                         unknown_t unk5, unknown_t unk6,
-                                         unknown_t unk7, unknown_t unk8) {
-  return X_ERROR_INVALID_PARAMETER;
+dword_result_t XamCreateEnumeratorHandle(dword_t user_index, dword_t app_id,
+                                         dword_t open_message,
+                                         dword_t close_message,
+                                         dword_t item_size, dword_t item_count,
```

Comment:
It's `extra_size` because there's a base size it's added to. Please don't rename the argument.

https://github.com/xenia-project/xenia/blob/f6f524b8148145c5b5b5ac0a79809df979069bc6/src/xenia/kernel/xenumerator.cc#L23-L27

Reply:
I renamed it after a few calculation based on PGR4 disassembly, but sure I can revert that change

Reply:
It's not the size of an item, but the extra size that gets added to the native enumerator object for that type enumerator specific data.